docs/tutorials/04-pipeline.ipynb

```diff
@@ -666,8 +666,7 @@
     "\n",
     "Congratulations! You’ve completed Part 4 of the MLRun getting-started tutorial. To continue, proceed to [**Part 5 Model monitoring**](05-model-monitoring.ipynb).\n",
     "\n",
-    "You might also want to explore the following demos:\n",
-    "- For an example of distributed training pipeline using TensorFlow, Keras, and PyTorch, see the [**mask detection demo**](https://github.com/mlrun/demo-mask-detection).\n",
+    "You might also want to explore the following demo:\n",
     "- To learn about using the feature store to process raw transactions and events in real-time and respond and block transactions before they occur, see the [**Fraud prevention demo**](https://github.com/mlrun/demo-fraud).  "
```

Comment:
this reference to feature-store demo comes out-of-the-blue here.
Better place is at the end of docs/feature-store/basic-demo.ipynb